### ISSUE TITLE
Build time

### ISSUE THREAD

**Issue by wayneashleyberry** [gobuffalo/pop#91]
I'm new to using `pop` and was wondering if there's a way to speed up build time. I've seen an old issue about [using build tags to exclude unneeded database support](https://github.com/markbates/pop/issues/27) but unsure if it was ever implemented, or how it would work.

I'm only using pop for migrations and it takes my build times from ~1s to ~30s. I need to ship a single binary and run migrations on application startup in certain cases, so I can't necessarily use `soda`.

Take this simple example:

```go
package main

import (
	"github.com/gobuffalo/packr"
	"github.com/gobuffalo/pop"
)

func main() {
	deets := &pop.ConnectionDetails{
		//
	}

	conn, err := pop.NewConnection(deets)
	must(err)

	box := packr.NewBox(Path)
	migrationBox, err := pop.NewMigrationBox(box, conn)
	must(err)

	migrationBox.Up()
}
```

```sh
$ go clean -cache; time go build main.go
       29.03 real        32.67 user         2.18 sys
```

**Comment by markbates** [comment]
That flag was removed in v0.11.0 and SQLite became opt-in instead of opt-out. Upgrade and you’re good. https://blog.gobuffalo.io/buffalo-v0-11-0-released-f7f79be826ff

**Comment by markbates** [comment]
It’s the same with pop, that’s where the flag is. You’re running an older version, or potentially, markbates/pop. Upgrade and you’ll be good.
